src/aws/config.rs

```diff
@@ -79,6 +79,12 @@ pub struct Config {
     /// - profile config: `role_session_name`
     /// - default to `reqsign`.
     pub role_session_name: String,
+    /// `duration_seconds` value will be load from:
+    ///
+    /// - this field if it's `is_some`.
+    /// - profile config: `duration_seconds`
+    /// - default to `3600`.
+    pub duration_seconds: Option<i32>,
```

Comment:
How about using `usize` instead? It's my personal preference in such cases.

Reply:
Yeah that makes sense. Updated